amplipi/streams.py

```diff
@@ -257,80 +259,70 @@ def connect(self, src):
     os.system(f'cp {toml_template} {toml_useful}')
 
     # Input the proper values
-    self.metaport = 5030 + 2*src
+    self.connect_port = 4070 + 2*src
     with open(toml_useful, 'r') as TOML:
       data = TOML.read()
-      data = data.replace('AmpliPi_TEMPLATE', f'{self.name}')
-      data = data.replace("device = 'ch'", f"device = 'ch{src}'")
-      data = data.replace('5030', f'{self.metaport}')
+      data = data.replace('device_name_in_spotify_connect', f'{self.name}')
+      data = data.replace("alsa_audio_device", f"ch{src}")
+      data = data.replace('1234', f'{self.connect_port}')
```

Comment:
It would be awesome to expose `bitrate` and `autoplay` to the user if possible.

Reply:
Can we support higher bit rates?

Reply:
Let's do this in the future when we are able to better characterize utilization. We still need a 4th Spotify account!